Application/Setting/Univention/UniventionUser.php

```diff
@@ -0,0 +1,280 @@
+<?php
+namespace SPHERE\Application\Setting\Univention;
+
+use SPHERE\Application\Setting\Univention\Service\Entity\TblUnivention;
+use SPHERE\System\Extension\Repository\Debugger;
+
+class UniventionUser
+{
+    private $curlhandle;
+    private $server;
+    private $token;
+
+    public function __construct() {
+
+        if(($tblUnivention = Univention::useService()->getUnivention(TblUnivention::TYPE_VALUE_SERVER))){
+            $this->server = $tblUnivention->getValue();
+        }
+        if(($tblUnivention = Univention::useService()->getUnivention(TblUnivention::TYPE_VALUE_TOKEN))){
+            $this->token = $tblUnivention->getValue();
+        }
+        $this->curlhandle = curl_init();
+    }
+
+    /**
+     * @param      $name // Suche nach Mandanten Beispiel: "ref-"
+     * @param bool $fromFirstChar
+     *
+     * @return array|bool
+     */
+    public function getUserListByName($name, $fromFirstChar = true)
+    {
+
+        curl_reset($this->curlhandle);
+
+
+        if($fromFirstChar){
+            $Url = 'https://'.$this->server.'/v1/users/?name='.$name.'%2A';
+        } else {
+            $Url = 'https://'.$this->server.'/v1/users/?name=%2A'.$name.'%2A';
+        }
+
+        curl_setopt_array($this->curlhandle, array(
+            CURLOPT_URL => $Url,
+            CURLOPT_HTTPGET => TRUE,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [TRUE, FALSE and NULL must be lowercase; expected "true" but found "TRUE"](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)